README.md

```diff
@@ -2,23 +2,24 @@
 
 **WIP**: A proof of concept for a CSAF trusted provider, checker and aggregator.
 
-
 ## Setup
 
 - A recent version of **Go** (1.17+) should be installed. [Go installation](https://go.dev/doc/install)
 
 - Clone the repository `git clone https://github.com/csaf-poc/csaf_distribution.git `
 
 - Build Go components
- ``` bash
- cd csaf_distribution
- go build -v ./cmd/...
+
+```bash
+cd csaf_distribution
+go build -v ./cmd/...
```

Comment:
Maybe, we should explicitly create a `./bin` and put the executables in there? :thinking:

Reply:
I would keep it that way `./cmd/...` - it is common practice and bin is more where it ends up after installation (the best URL I could come up with using my current search-enging-foo is https://github.com/golang-standards/project-layout ...)